### PR TITLE
Nav revamp

### PR DESCRIPTION
**Pull request by jomarmontuya**

### CODE REVIEW

**Comment by jomarmontuya** [comment]
https://user-images.githubusercontent.com/61284357/204282370-ec9e4e53-296b-4693-b101-f7f02255df17.mp4

File: src/pages/marketplace/index.js [hunk 1]
```diff
@@ -18,26 +18,41 @@ import Head from 'next/head';
...
+
+const Marketplace = ({ marketEntities, marketEntityTypes, nav, ...props }) => {
   const seoTitle = props.meta.web.seo_meta_title,
     seoDescription = props.meta.web.seo_meta_description;
 
   useEffect(() => {
     setCookie('PRODUCTION', props.zestyProductionMode);
   }, [props]);
 
+  console.log(nav);
```
Comment (darwin808) [inline src/pages/marketplace/index.js]: remove clg

**Review by darwin808 — CHANGES_REQUESTED** [review]

**Review by darwin808 — DISMISSED** [review]

**Comment by jomarmontuya** [comment]
Holding off this navigation flyout till new pages are fully built out per katie

Sending back to draft

**Review by darwin808 — CHANGES_REQUESTED** [review]
error when production = true 

image

No nav item in production =false testing local

image

**Review by darwin808 — APPROVED** [review]

**Pull request merged**